packages/clis/parser/game-files/sii-visitors.ts

```diff
@@ -74,17 +74,32 @@ class JsonConverterVisitor extends getSiiVisitorClass<
   ): void {
     if (children.String) {
       json.value = quotedStringToString(children.String[0].image);
+    } else if (children.BinaryFloat) {
+      json.value = stringToFloat(children.BinaryFloat[0].image);
```

Comment:
Ahyes.

Incidentally, the reason why I got confused about the order is that the order of token definitions at the top of this file doesn't match that of `objectPropertyValue`.

https://github.com/truckermudgeon/maps/blob/885e0cf9992cfb54da95f0ecfe0f011762480299/packages/clis/parser/game-files/sii-parser.ts#L27-L41

Reply:
Ahhh... in that case, could you also re-arrange the order of token definitions to match that of `objectPropertyValue`?

Reply:
Actually, nevermind... you won't be able to do that because of declaration issues:

<img width="673" alt="image" src="https://github.com/user-attachments/assets/104214fa-6851-4bb0-84d4-04777880a547" />

Ignore my previous comment, and pay no attention to the ~man behind the curtain~ order of token definitions 😅.